lgensemble.go

```diff
@@ -37,6 +37,10 @@ func (e *lgEnsemble) Name() string {
 	return e.name
 }
 
+func (e *lgEnsemble) NLeaves() []int {
+	return GetNLeaves(e.Trees)
```

Comment:
Can't find GetNLeaves function in the code base..

Reply:
Sorry, forgot to push one commit.

Pushed just now